File: pygameavs/src/btpy/src/btpy/btpy_list/mod/clean_voids/clean_voids.py



def clean_voids(ARRAY)->list:
    """
     Función que crea un nuevo 
     eliminando todos los valores 
     None y void
    """
    new_array = []
    for i in range(len(ARRAY)):
        if(ARRAY[i] == None): continue
        if(ARRAY[i] == ""): continue
        if(ARRAY[i] == []): continue
        if(ARRAY[i] == {}): continue
        new_array.append(ARRAY[i])
    return new_array

File: btjs_page/btjs/mod/structures/mod/clean_voids.js



/*
Función que crea un nuevo array
eliminando todos los valores 
None y void
*/
export function clean_voids(ARRAY) {
    new_array = []
    for(let i in ARRAY) {
        if(ARRAY[i] == null) {continue;}
        if(ARRAY[i] == "") {continue;}
        if(ARRAY[i] == []) {continue;}
        if(ARRAY[i] == {}) {continue;}
        new_array.append(ARRAY[i])
    }
    return new_array
}